app/models/rental_creator.rb

```diff
@@ -0,0 +1,8 @@
+class RentalCreator
+  def self.create_rental(cart, session)
+    rental = Rental.create(user_id: session[:user_id], days_rented: session[:cart]['days'], total_price: cart.total_price, status: 'Paid')
```

Comment:
Line is too long. [138/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.